includes/hub/database/class-orders.php

```diff
@@ -80,6 +80,9 @@ public static function register_post_type() {
 			'show_in_menu'     => Network_Admin::PAGE_SLUG,
 			'can_export'       => false,
 			'capability_type'  => 'post',
+			'capabilities'     => [
+				'create_posts' => false, // Removes "add new".
```

Comment:
NIT, but even though this end up working, it's not completely correct... The idea here is to inform a string with which capability will be checked to see if a user can create a post in this post type. By default it will check `create_posts` , we can change the suffix in `capabaility_type` for all caps but we are not.

What we usually do when we want to block something for everybody is to return a capability that noboby will ever have. Like "not_allow" or something.